src/gf-fsck

```diff
@@ -19,9 +19,13 @@ boot=$(coreos_gf findfs-label boot)
 coreos_gf debug sh "fsck.ext4 -f -n ${boot}"
 
 # ESP, if it exists
-if [ "$(coreos_gf part-get-gpt-type /dev/sda 1)" = "${EFI_SYSTEM_PARTITION_GUID}" ]; then
-    coreos_gf debug sh "fsck.vfat -f -n /dev/sda1"
-fi
+local partitions="$(coreos_gf list-partitions)"
+for pt in $partitions; do
+    local label="$(coreos_gf vfs-label $pt)"
+    if [ "$label" == "EFI-SYSTEM" ]; then
+        coreos_gf debug sh "fsck.vfat -f -n $pt "
+    fi
+done
```

Comment:
Could this be replaced with something like

```suggestion
efi=$(coreos_gf findfs-label EFI-SYSTEM || :)
if [ -n "${efi}" ]; then
    coreos_gf debug sh "fsck.vfat -f -n $efi"
done
```

?

Reply:
I tried locally, but it fails on `s390x`. Despite search should fail, it fails with `EXIT_FAILURE` (or smth similar) and `guestfish` dies

Reply:
So i wasn't able to find something better than `for` loop